acceptance/tests/api/value/core_and_custom.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+test_name 'Facter.value(core_fact) when custom fact is defined' do
```

Comment:
should probably differentiate between this test's title and the following one from `core_and_custom_overwrite.rb` as they have the same name